scripts/check-component-features

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -o errexit
+
+# Extract all feature names from Cargo.toml
+features=$(
```

Comment:
I have a slight preference for just listing out the excluded features by name to avoid it being greedier than we want.

Reply:
By "greedier" do you mean having it skip too many features or something else?

Reply:
Ah, yeah, that it would skip features we actually want to test.

Reply:
Are there `*-utils-*` features we do want to test separately? The other two features we skip are listed explicitly.

Reply:
Not currently, but I could see us adding one. I think the only reason we ignore them currently is that it is too difficult to untangle. Not a big deal regardless, though. I'm happy with it as-is.